components/user_groups_modal/user_groups_modal.scss

```diff
@@ -0,0 +1,347 @@
+@charset 'UTF-8';
+
+.modal {
```

Comment:
Again, this is not scoped to a component class.

Reply:
Root class is now related to the component name